types/recharts/index.d.ts

```diff
@@ -540,8 +545,9 @@ export interface ReferenceAreaProps {
 	viewBox?: ViewBox;
 	xAxis?: any;
 	yAxis?: any;
-	label?: string | number | React.ReactElement<any> | RechartsFunction;
+	label?: string | number | React.ReactElement<any> | ContentRenderer<ReferenceAreaProps>;
 	isFront?: boolean;
+    className?: string;
```

Comment:
I'm not sure about this. But aren't `className`, `style` and other props part of `React.ReactElement` props? In my opinion, they should come from somewhere else e.g. extend another interface or class.

Reply:
They are part of HTMLAttributes but recharts doesn't allow all HTMLAttributes. We can leave them out as they are not mentioned in the docs either.

Reply:
yeah perfect. @Metavirulent please remove this one also (line 550) I think you missed this one in your latest commit.